packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2764,6 +2764,15 @@ export default class Meeting extends StatelessWebexPlugin {
       this.recordingController.setUserPolicy(this.selfUserPolicies);
       this.controlsOptionsManager.setDisplayHints(this.userDisplayHints);
       this.handleDataChannelUrlChange(this.datachannelUrl);
+
+      Trigger.trigger(
```

Comment:
`LOCUSINFO.EVENTS.MEETING_INFO_UPDATED` is emitted from `updateMeetingInfo()`, which is called also from `updateLocusInfo()` but also from `init()` in locus-info/index.ts. Can you manually check that this won't trigger this new event you're adding also every time when a meeting is created? It might be also worth adding that check to some existing unit tests that create meetings.